src/components/item-list.tsx

```diff
@@ -18,68 +20,88 @@ export const ItemList = ({
   focusQuests,
   omittedItems,
 }: Props) => {
-  return (
-    <div>
-      <div className="search-wrapper">
-        <input
-          type="text"
-          onChange={(e) => setSearch(e.target.value)}
-          placeholder="Search for items"
-        />
-      </div>
-      <div>💀🗑️ = Item must be dead dropped</div>
-      <hr />
+  const searchCount = search.length;
+  const itemsNeededCount = Object.keys(itemsNeeded).length;
+  const itemsCompletedCount = Object.keys(itemsNeeded).filter((key) => itemsNeeded[key as ItemName] === 0).length;
+  const focusQuestsCount = focusQuests.length;
 
-      {omittedItems === "quest" ? (
-        <div>Items needed for quests are currently hidden.</div>
-      ) : focusQuests.length > 0 ? (
-        <div>
-          Only showing items from:
-          <ul className="quest-list">
-            {focusQuests.map((questName) => (
-              <li key={questName}>
-                {questName} - part {questProgress[questName] + 1}
-              </li>
-            ))}
-          </ul>
-        </div>
-      ) : null}
-      {omittedItems === "upgrade" ? (
-        <div>Items needed for upgrades are currently hidden.</div>
-      ) : null}
-      {omittedItems ||
-        (omittedItems === "quest" && focusQuests.length > 0 && <hr />)}
+  const searchBox = (<div className="search-wrapper">
+    <input
+      type="text"
+      onChange={(e) => setSearch(e.target.value)}
+      placeholder="Search for items"
+    />
+  </div>);
+
+  const sellableList = (<div>
+      <hr />
+      <div>💰 Sellable Items</div>
+      <hr />
       {Object.keys(itemsNeeded)
-        .filter((key) => itemsNeeded[key as ItemName] > 0)
+        .filter((key) => itemsNeeded[key as ItemName] === 0)
         .sort()
         .map((key) => {
-          if (
-            search.length === 0 ||
-            key.toLocaleLowerCase().includes(search.toLocaleLowerCase())
-          ) {
-            return (
-              <div key={key}>
-                {key}: {itemsNeeded[key as ItemName]}{" "}
-                {quests
-                  .filter(
-                    (quest) => quest.parts.length > questProgress[quest.name]
-                  )
-                  .some((quest) =>
-                    quest.parts.some((part) =>
-                      part.dropItems?.some((item) => item.item === key)
-                    )
-                  )
-                  ? "💀🗑️"
-                  : null}
-              </div>
-            );
+          if (searchCount === 0 || key.toLocaleLowerCase().includes(search.toLocaleLowerCase())) {
```

Comment:
I would prefer to just see search.length here, I think searchCount makes it sound like it's the quantity of items returned by a search or somesuch. Caused me some cognitive load trying to figure out what it is.